tests/unit/broccoli/ember-addon-test.js

```diff
@@ -7,6 +7,18 @@ const EmberApp = require('../../../lib/broccoli/ember-app');
 const { expect } = require('chai');
 const MockCLI = require('../../helpers/mock-cli');
 
+const EMBER_SOURCE_ADDON = {
```

Comment:
I've repeated this a couple of times, simply because I couldn't find a decent sport to centralise this 😬.